lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.reef.runtime.local.driver;
 
+import com.google.common.primitives.Ints;
```

Comment:
It [safely casts] (https://stackoverflow.com/questions/1590831/safely-casting-long-to-int-in-java) the long to int. Otherwise, there would be the warning.